### DISCUSSION TITLE
Awesome!

### DISCUSSION THREAD

**Discussion by brees1337** [FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher#95] [General]
Just wanted to say that this worked perfectly, allows me to relish in sims1 nostalgia in perfect 1080p resolution.  Thank you to everyone that made this possible! :)

**Comment by FaithBeam** [comment]
There are so many people that have helped directly and indirectly over the years with this project, it's awe-inspiring. As the saying goes, "It takes a village". 
I'm glad you enjoy it.

**Reply by rodrigoaguilar11** [reply]
Hello FaitBeam, thank you very much for your work in the Sims 1 community, it is really very important, I currently have the largest community of the game in Spanish and I am spreading your patch to the followers.

**Reply by FaithBeam** [reply]
Awesome! Thank you for sticking around, and I hope your followers find it useful :)
